qutebrowser/browser/network/networkmanager.py

```diff
@@ -358,6 +358,23 @@ def createRequest(self, op, req, outgoing_data):
             dnt = '0'.encode('ascii')
         req.setRawHeader('DNT'.encode('ascii'), dnt)
         req.setRawHeader('X-Do-Not-Track'.encode('ascii'), dnt)
+
+        def same_domain():
+            tabbed_browser = objreg.get('tabbed-browser', scope='window', window=self._win_id)
+            view = tabbed_browser.currentWidget()
+            print(req.url().host() == view.url().host(), repr(view.url().host()), repr(req.url().host()))
+            # TODO: We probably want to allow headers if we're on test.com and
+            # doing a request to www.test.com? Or maybe a new settings value for
+            # this?
+            return req.url().host() == view.url().host()
```

Comment:
I'll just use  `QUrl::topLevelDomain` for now. Replacing it with something else should not be difficult.